crates/matrix-sdk-ui/src/timeline/controller/mod.rs

```diff
@@ -173,10 +181,11 @@ impl Default for TimelineSettings {
     }
 }
 
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone)]
```

Comment:
I think this can still be `Copy`, and then you can avoid adding `.clone()` everytime this is copied :)

Reply:
Ah sorry, this was left over from the other branch where I introduced an OwnedEventId as an associated value on the Thread here.